nifi-nar-bundles/nifi-websocket-bundle/nifi-websocket-services-jetty/src/main/java/org/apache/nifi/websocket/jetty/JettyWebSocketClient.java

```diff
@@ -219,7 +219,7 @@ public void startClient(final ConfigurationContext context) throws Exception{
                 throw new IllegalArgumentException(AUTH_CHARSET.getDisplayName() + " was not specified.");
             }
             final Charset charset = Charset.forName(charsetName);
-            final String base64String = Base64.encodeBase64String((userName + ":" + userPassword).getBytes(charset));
+            final String base64String = Base64.getEncoder().encodeToString((userName + ":" + userPassword).getBytes(charset));
```

Comment:
Does this eliminate org.apache.commons as a dependency or is there another reason for this?

Reply:
The `nifi-websocket-services-jetty` library had a dependency on `nifi-security-utils`, which transitively included Apache `commons-codec`.  With the removal of `nifi-security-utils`, this change removes the need for `commons-codec` in this module by using the JDK Base64 Encoder.